### ISSUE TITLE
[Bug]: [A11y] Breadcrumb Component: Slash Separators Are Not Hidden from Screen Readers

### ISSUE THREAD

**Issue by ravindar26** [SAP/fundamental-styles#6212]
### Describe the bug

**Description:**

**Issue Summary:**  
In the breadcrumb component, slash separators intended for visual separation between breadcrumb links are not properly hidden from screen readers. This behavior is inconsistent with accessibility best practices, which aim to enhance the user experience for individuals relying on assistive technologies.

**Affected Components:**  
- [SAP Fiori Breadcrumb Component](https://sap.github.io/fundamental-styles/?path=/docs/sap-fiori-components-breadcrumb--docs)

**Steps to Reproduce:**  
1. Navigate to a page implementing the breadcrumb component as described in the SAP Fiori documentation.
2. Utilize the JAWS screen reader to traverse the breadcrumb links.
3. Notice that the slash separators, added via CSS content, are not ignored by the screen reader.

**Expected Behavior:**  
The slash separators should be hidden from screen readers to ensure a clean and logical reading order. An appropriate method should be in place for this component to hide non-essential visual elements from assistive technologies.

**Actual Behavior:**  
Slash separators introduced via CSS content are read by screen readers, potentially causing confusion or a disrupted user experience.

**Impact:**  
This issue may lead to a reduced usability experience for users relying on screen readers, as extraneous content may be read aloud to them, causing potential confusion.

**Proposed Solution:**  
Investigate potential methods to ensure that slash separators added through CSS content can be excluded from screen readers. This might involve adjusting the approach to include the separators as DOM elements with `aria-hidden` attributes, or another similar solution that complies with accessibility standards.

**References and Additional Information:**  
- **UI5 Implementation:** [UI5 Breadcrumbs](https://ui5.sap.com/#/entity/sap.m.Breadcrumbs/sample/sap.m.sample.Breadcrumbs)
- **Web Components Implementation:** [Web Components Breadcrumbs](https://ui5.github.io/webcomponents/components/Breadcrumbs/)
- **Screen Reader:** JAWS

**Screenshot**

<img width="3780" height="2238" alt="Image" src="https://github.com/user-attachments/assets/ce7e7a65-cdb2-4043-bc6d-17cd4d49c461" />

### Fundamental Library Styles Version

0.40.1

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by droshev** [comment]
@InnaAtanasova do you think you will be able to find a solution?
